docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -120,6 +120,12 @@ Allows setting a sort order for the result. Defaults to `start_time`, i.e. sorti
 
 `index_count`::
   Sort snapshots by the number of indices they contain and break ties by snapshot name.
+
+`shard_count`::
```

Comment:
++ that makes good sense, I made the external parameter `failed_shard_count` now to make it consistent. I think the enum/internal names are ok as is and we wouldn't really want to change them at this point for BwC reasons (at least technically the transport client would have its API broken otherwise).